modules/articles/client/views/form-article.client.view.html

```diff
@@ -13,8 +13,8 @@
           </div>
         </div>
         <div class="form-group">
-          <label class="control-label" for="content">Content</label>
-          <textarea name="content" data-ng-model="vm.article.content" id="content" class="form-control" cols="30" rows="10" placeholder="Content"></textarea>
+          <label for="content">Content</label>
+          <summernote name="content" config="summeroptions" ng-model="vm.article.content" id="content"></summernote>
```

Comment:
Agree with @ilanbiala on the UI effort argument. Although Ilan, if this is just minor addition and not really something that will introduce problems/maintenance overhead then I don't mind.